pkg/services/ngalert/state/manager_test.go

```diff
@@ -1451,7 +1457,11 @@ func printAllAnnotations(annos map[int64]annotations.Item) string {
 }
 
 func TestIntegrationStaleResultsHandler(t *testing.T) {
-	evaluationTime := time.Now().Truncate(time.Second).UTC() // Truncate to the second since we don't store sub-second precision.
+	if testing.Short() {
+		t.Skip("skipping integration test in short mode")
+		// Truncate to the second since we don't store sub-second precision.
+	}
+	evaluationTime := time.Now().Truncate(time.Second).UTC()
```

Comment:
```suggestion
	}

	evaluationTime := time.Now().Truncate(time.Second).UTC() // Truncate to the second since we don't store sub-second precision.
```